README.md

```diff
@@ -19,22 +19,22 @@ Using Maven:
 See more examples in the `./samples` folder.
 
 ```java
-import com.coveo.pushapiclient.DocumentBuilder;
-import com.coveo.pushapiclient.Source;
+import com.coveo.document.DocumentBuilder;
+import com.coveo.source.SourceClient;
 
 import java.io.IOException;
 import java.net.http.HttpResponse;
 
 public class PushOneDocument {
     public static void main(String[] args) {
-        Source source = new Source("my_api_key", "my_org_id");
+        SourceClient sourceClient = new SourceClient("my_api_key", "my_org_id");
```

Comment:
Same question here, this feels like a breaking change? Is it intentional? If so, is it necessary?

Reply:
same reply as above